content/en/tracing/trace_collection/library_config/_index.md

```diff
@@ -3,14 +3,117 @@ title: Configure the Datadog Tracing Library
 type: multi-code-lang
 ---
 
-For information about configuration options for tracing libraries, choose your language:
+This page describes configuration options that behave consistently across all languages. To view these common configuration options, see [Common configuration options](#common-configuration-options).
+
+For configuration options specific to your programming language, choose your language from the options below:
 
 {{< partial name="apm/apm-compatibility.html" >}}
 
 <br>
 
 To instrument an application written in a language that does not yet have official library support, see the list of [community tracing libraries][1].
 
+## Common configuration options
+The following configuration options behave consistently across the latest versions of all Datadog SDKs, unless otherwise noted:
+
+### Traces
+
+`DD_TRACE_<INTEGRATION>_ENABLED`
+: **Default**: `true` <br>
+**Supported Input**: Boolean <br>
+**Caveats**: Not supported in Go & in Java integrations labeled as "beta" or marked as "Disabled" in this document are disabled by default.<br>
```

Comment:
This is a bit hard to read as-is. Consider having it as a bulleted list instead
```suggestion
**Caveats**:<br>
- Not supported in Go
- [Some Java integrations are disabled by default][2].
```

Where [2] links to https://docs.datadoghq.com/tracing/trace_collection/compatibility/java/#framework-integrations-disabled-by-default